pkg/controller/job/jobcontroller.go

```diff
@@ -135,39 +117,26 @@ func NewJobController(podInformer cache.SharedIndexInformer, kubeClient clientse
 	return jm
 }
 
-func NewJobControllerFromClient(kubeClient clientset.Interface, resyncPeriod controller.ResyncPeriodFunc) *JobController {
```

Comment:
Was about to say that. @deads2k added that when [switching to shared informer](https://github.com/kubernetes/kubernetes/commit/8c4e3af1), it's used only in tests, nowhere else.